src/orchard/cljs/analysis.cljc

```diff
@@ -0,0 +1,203 @@
+(ns orchard.cljs.analysis
+  (:require [orchard.misc :as u]
+            #?(:cljs [orchard.cljs.special :as special]))
+  (:refer-clojure :exclude [find-ns find-var all-ns ns-aliases]))
+
+(def NSES :cljs.analyzer/namespaces)
+
+(defn all-ns
```

Comment:
I guess you should also implement `orchard.query` based on this.

Reply:
yeah that's what I was asking yesterday, not really familiar with that namespace and it deserves a new PR I guess